### DISCUSSION TITLE
Github Classroom with Git LFS

### DISCUSSION THREAD

**Discussion by malcolmryan** [community/community#54070] [GitHub Education]
### Hi everyone,

It looks like there has been a new policy implemented that does not allow repositories with Git LFS content to be used as Template repositories. This means that Github Classroom repos can no longer use LFS. Teaching game development, our assignments typically contain a lot of binary files and we teach LFS as the proper way to handle them. This change is going to make it difficult for us to run these assignments. 

Has anyone else encountered this issue? Are there any workarounds?

**Comment by bitboyb** [comment]
Sorry to comment after a long time that this was initially posted. This is an issue I have encountered. It would be vital to support LFS in GitHub Classroom. Sadly, we just get them to set up their own repositories or we just provide a repository they can fork but that is incredibly messy. If they implemented forks as an option instead of templates. It would make it excellent for teaching games.
